src/test/kotlin/org/rust/ide/inspections/lints/RsRedundantSemicolonsInspectionTest.kt

```diff
@@ -0,0 +1,64 @@
+/*
+ * Use of this source code is governed by the MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package org.rust.ide.inspections.lints
+
+import org.rust.ide.inspections.RsInspectionsTestBase
+
+class RsRedundantSemicolonsInspectionTest : RsInspectionsTestBase(RsRedundantSemicolonsInspection::class) {
```

Comment:
I found a scenario when the quick-fix breaks the code (from https://github.com/rust-lang/rust/blob/master/src/test/ui/lint/redundant-semicolon/item-stmt-semi.rs)
```Rust
fn main() {
    fn inner() {};/*<caret>*/
    struct Bar {};
}
```
after invoking the quick-fix
```Rust
fn main() {
    fn inner() {}
}
```

Reply:
Oh, tnx! It turns out items are not considered `RsStmt`s atm. Added a workaround for this PR but will probably try to properly fix this in another PR